care/facility/tests/test_patient_api.py

```diff
@@ -98,6 +99,28 @@ def _get_contact_patients_representation(self, patient: PatientRegistration):
             for contacted_patient in patient.contacted_patients.all()
         ]
 
+    @classmethod
+    def _change_dob(cls, val: datetime.date):
+        """
+        Add a random number to day for the date of birth\
+        This is done to bypass the uniqueness of date of birth and phone number
+
+        Returns:
+            datetime.date
+                the modified date of birth
+        """
+        return val + datetime.timedelta(days=random.randrange(1, 365))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5359892)